ivy/functional/frontends/numpy/statistics/order_statistics.py

```diff
@@ -48,6 +48,68 @@ def ptp(a, axis=None, out=None, keepdims=False):
     return ret.astype(a.dtype, copy=False)
 
 
+@handle_numpy_out
+def percentile(
+    a,
+    q,
+    axis=None,
+    out=None,
+    overwrite_input=False,
+    method="linear",
+    keepdims=False,
+    interpolation=None,

```

Comment:
This is a keyword-only arg according to the [docs](https://numpy.org/doc/stable/reference/generated/numpy.percentile.html)